src/env_sim/launch/stage.launch

```diff
@@ -18,15 +17,6 @@
 	<node pkg="slam_mode_goal" type="slam_mode_goal.py" name="slam_mode_goal" output="screen">
 	</node>
 
-
-	<!-- VISUALIZATION -->
-	<node name="rviz" pkg="rviz" type="rviz" />
```

Comment:
Why did you remove this?

Reply:
Rviz was not being used. It was only launched and it did not display anything.

Reply:
So with RViz in this launch file, there was no window that popped up with a visualization of the kart going around the 2D track? If so, that is really weird and I'll take a look. But if it did show up, then I think we should keep this as we want to see the visualization.

Reply:
A window does pop up, but it does not display anything. Here's a screenshot of it. It is not being used by anything and only the window to the left is useful, and it is controlled by stage_ros.

![Screenshot from 2022-02-15 18-09-53](https://user-images.githubusercontent.com/65411471/154165309-15d9e797-f716-4b16-9e48-e90c4739ae2d.png)